src/main/java/com/dougnoel/sentinel/elements/dropdowns/SelectElement.java

```diff
@@ -122,4 +122,13 @@ public boolean doesNotHaveOption(String text) {
 				.ignoring(StaleElementReferenceException.class)
 				.until(d -> selectElement.getOptions().stream().noneMatch(el -> el.getText().endsWith(text)));
     }
+    
+    /**
+     * Returns the number of options in this Select element.
+     * @return int the number of options in this Select element.
+     */
+    public int getNumberOfOptions() {
+    	Select selectElement = new Select(this.element());
+    	return selectElement.getOptions().size();
```

Comment:
I may be over-optimizing, but we don't actually need the variable declaration.
```suggestion
    	return new Select(this.element()).getOptions().size();
```